samples/java_springboot/21.corebot-app-insights/src/main/java/com/microsoft/bot/sample/corebot/app/insights/CancelAndHelpDialog.java

```diff
@@ -19,8 +19,8 @@
  */
 public class CancelAndHelpDialog extends ComponentDialog {
 
-    private final String helpMsgText = "Show help here";
-    private final String cancelMsgText = "Cancelling...";
+    private static String helpMsgText = "Show help here";
+    private static String cancelMsgText = "Cancelling...";
```

Comment:
Same, rollback this (see [C#](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/21.corebot-app-insights/Dialogs/CancelAndHelpDialog.cs#L14-L15))